src/lang-pt_BR.js

```diff
@@ -0,0 +1,517 @@
+    minoption: 'Inserir pelo menos duas opções',
+    microphoneNotConnected: 'Microfone não está disponível, ou não está devidamente conectado',
+    // 'mictesting' : 'Se a barra de áudio acima oscilar enquanto fala, seu microfone está funcionando.',
+    mictesting: 'Seu microfone está funcionando se a barra de áudio acima oscilar <br />'
+    + 'e você conseguir ouvir sua própria voz, enquanto fala',
+    minute: 'Minuto',
+    ModeclosingPoll: 'Modo de fechamento de Enquete :',
+    msgForDownloadStart: 'Incapaz de salvar os dados. <br /> Preparando dados para download',
+    msgForReload: 'Favor recarregar esta página para continuar editando.',
+    msgForWhiteboard: 'Quadro branco vazio.',
+    msgStudentForReload: 'Favor recarregar esta página.',
+    Multiple_login: 'Erro: Acesso negado, múltiplos logins',
+    muteAll: 'Mutar todos',
+    muteAllAudio: 'Mutar todo áudio',
+    mybsharedoc: 'Documento será compartilhado logo',
+    NAME: 'NOME',
+    Next: 'Próximo',
+    note: 'Adicionar anotação',
+    nomdlroot: 'Não há url para a raíz momodle.',
+    normalView: 'Visualização normal',
+    noResultStd: 'Você não tem permissão para ver o resultado',
+    NotAllowedError: 'Webcam está desabilitada',
+    NotFoundError: 'Webcam não está disponível, ou não está devidamente conectada.',
+    NotReadableError: 'Favor certifique-se que a mesma webcam não está sendo usada <br /> em múltiplos browsers ou múltiplas aplicações.',
+    notsupportbrowser: '{virtualclass1} não é completamente compatível. Para a melhor experiência, use o Google Chrome.',
+    NotSupportedError: 'Somente origens seguras(https) são permitidas para áudio e vídeo',
+    notSupportIphone: "Desculpe. A sala virtual não suporta smartphones.",
+    noPoll: 'Não há nenhuma questão disponível para a enquete. Vá em ‘Adicionar novo’ para criar uma questão.',
+    noPollNoAdmin: 'Não há nenhuma questão disponível para a enquete de site. Só o admin pode criar enquete a nível de site.',
+    noQuiz: `Não há nenhum Quiz disponível. Adicione um quiz do seu Moodle.
+    Congrea suporta apenas questões de múltipla escolha.`,
+    Novote: 'Nenhum voto recebido para esta enquete',
+    nowebcam: 'Webcam não está disponível, ou não está devidamente conectada.',
+    nowebcamconnectedyet: 'Congrea não recebeu nenhuma entrada do usuário ainda.',
+    notcompatiblecpu: 'Poder de processamento (CPU) no sistema é baixo. <br /> Nós recomendamos usar uma CPU moderna.',
+    notcompatibleram: 'RAM no sistema é baixo. <br /> Nós recomendamos 4GB de RAM.',
+    offcollaboration: 'Colaboração desligada',
+    oncollaboration: 'Colaboração ligada',
+    opendsDbheading: 'Painel de documento',
+    openSharePresentationdbHeading: 'Painel de apresentação',
+    openvideoDashboard: 'Painel de vídeo',
+    operaBrowserIssue: 'Seu browser {virtualclass1} {virtualclass2} é parcialmente suportado. Você não poderá compartilhar sua tela. Congrea suporta completamente Chrome e Firefox',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This line has a length of 181. Maximum allowed is 120.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5482325)